src/main/cpp/t07_ip.cpp

```diff
@@ -29,10 +29,35 @@
 
 #include "t07_ip.h"
 #include <iostream>
+#include <cstring>
 
 
 using namespace std;
 
 int t07_ip() {
-
+	string h;
+	getline(cin, h);
+	int m1 = 0, m2 = 0, m3 = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'm3' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4578786)